test/langtools/tools/javac/patterns/BindingsTest2.out

```diff
@@ -49,6 +49,6 @@ BindingsTest2.java:241:17: compiler.err.cant.resolve.location: kindname.variable
 BindingsTest2.java:135:17: compiler.err.unreachable.stmt
 BindingsTest2.java:160:17: compiler.err.unreachable.stmt
 BindingsTest2.java:185:17: compiler.err.unreachable.stmt
-- compiler.note.preview.filename: BindingsTest2.java
+- compiler.note.preview.filename: BindingsTest2.java, DEFAULT
 - compiler.note.preview.recompile
 51 errors
```

Comment:
(issue unrelated to review)  it seems wrong that we encourage/allow the use of files that do not end in newline.